app/javascript/components/AddReservationForm.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const AddReservationForm = ({ onAddReservation }) => {
```

Comment:
- [ ] When I add a reservation and try to view it in "my reservations", I get greeted with a blank screen:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/661859b0-24aa-428a-a32a-1f9863368df0)

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/08d55079-84b4-4230-9e89-0092375c39e8)

Since the server does not do anything when doing this, I am inferring this is a frontend problem.

The only way to get into "my reservations" after adding a reservation, is by reloading the page and logging in again:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/379bb6f1-2173-46b7-93dd-1c8a36a93a32)

Kindly, fix this bug so your app can show the reservations after immediately adding one without having to reload the page, as intended. 👍